packages/builder/src/builtin-tasks/compile.ts

```diff
@@ -0,0 +1,72 @@
+import { task } from "../internal/core/config/config-env";
+import { networks } from "../../sample-project/algob.config";
+import { TASK_COMPILE } from "./task-names";
+import YAML from "yaml";
+const algosdk = require('algosdk');
+const fs = require('fs');
+const path = require('path');
+const murmurhash = require('murmurhash');
+
+/* Credentials */
```

Comment:
I told you that if you can't use the config you should hardcode the variables. You shouldn't import `sample-project` in any circumstance. This is not the use of the `sample-project`. It mustn't be used in `compile` task. You have a local project that you have to change, not the default one.